ClosedXML/Extensions.cs

```diff
@@ -354,4 +359,41 @@ public static bool IsNumber(this object value)
                     || value is decimal;
         }
     }
+
+    public static class RangeExtensions

```

Comment:
I did not want to change the interface IXLRangeBase. Do you think it's better to add new methods into interface?